src/renderers/dom/client/wrappers/ReactDOMInput.js

```diff
@@ -213,7 +213,11 @@ var ReactDOMInput = {
       }
     } else {
       if (props.value == null && props.defaultValue != null) {
-        node.defaultValue = '' + props.defaultValue;
+        // Assigning defaultValue causes side-effects on value, which can cause
```

Comment:
Can you please rephrase this comment? It's not clear what you mean exactly to a person new to this.
A link to the issue would also be most helpful.